### DISCUSSION TITLE
60000 bind variables?! Maybe it’s time to use a temporary table instead… | Tanel Poder Consulting

### DISCUSSION THREAD

**Discussion by giscus** [tanelpoder/blog-comments#37] [Announcements]
# 60000 bind variables?! Maybe it’s time to use a temporary table instead… | Tanel Poder Consulting

I just noticed a bug 8277300 filed in Metalink with following description:
ORA-7445[XTYQBCB] OCCURS DURING EXECUTING SQL THAT USES 60000 BIND VARIABLES. Wow! That’s about 100 times more bind variables in a single query than what I’ve seen in past. And I thought that query was bad!!! :)
I suspect this is a massive IN list passed to a query. Maybe it’s time to use a temporary table or a collection for passing in the IN values instead? - Linux, Oracle, SQL performance tuning and troubleshooting - consulting & training.

http://localhost:1313/2009/02/25/60000-bind-variables-maybe-its-time-to-use-a-temporary-table-instead/

**Comment by tanelpoder** [comment]
test
